crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort_config.rs

```diff
@@ -7,17 +7,29 @@
 
 use std::collections::HashMap;
 
+use bitvec::{order::Lsb0, vec::BitVec};
+
+use super::presets::ConfigPreset;
+
 /// A utility layer, containing its name and an ordered list of classes.
 pub struct UtilityLayer {
     pub name: &'static str,
     pub classes: &'static [&'static str],
 }
 
+pub fn build_variant_weight(size: usize) -> BitVec<u8, Lsb0> {
+    let mut bit_vec = BitVec::new();
+    let iterable = vec![false; size];
+    bit_vec.extend(iterable);
+    bit_vec.push(true);
```

Comment:
Why do we push a `true` at end?

Reply:
> Why do we push a `true` at end?

This the `1` bit of the BitVec representing a variant weight, it's ordered by Lsb0 so it will look like `[0,0,0,0,1]` for a variant that has index 4.
Weights are all in this form.